README.md

```diff
@@ -49,7 +49,15 @@ nnoremap <F5> :UndotreeToggle<CR>
 
 ```
 if has("persistent_undo")
-    set undodir=$HOME."/.undodir"
+   let target_path = expand('~/.undodir')
+
+    " create the directory and any parent directories
+    " if the location does not exist.
+    if !isdirectory(target_path)
+        call system('mkdir -p ' . target_path)
```

Comment:
using `:call mkdir(target_path, "p", 0700)` would be more compatible since some OS might not have the `mkdir` command.

Reply:
Sweet.
Thanks.
Fxd